### PR TITLE
SHARD-990: verifyGlobalTxReceipt unit tests

### PR DESCRIPTION
**Pull request by S0naliThakur**
Linear: https://linear.app/shm/issue/SHARD-990
Summary: unit tests for verifyGlobalTxReceipt method

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +49,7 @@
   },
   "homepage": "https://gitlab.com/shardus/archive/archive-server#readme",
   "devDependencies": {
+    "@types/jest": "^29.5.14",
```
Comment (aniketdivekar) [inline package.json]: let's please use fixed versions

File: src/dbstore/accounts.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export type AccountsCopy = {
   data: any // eslint-disable-line @typescript-eslint/no-explicit-any
   timestamp: number
   hash: string
-  cycleNumber: number
+  cycleNumber?: number
```
Comment (aniketdivekar) [inline src/dbstore/accounts.ts:13]: does this have to be optional?

Comment (S0naliThakur) [reply src/dbstore/accounts.ts:13]: yes. not all AccountsCopy objects have cycleNumber

File: package.json (hunk 1 above)
Comment (S0naliThakur) [reply package.json]: done

**Review by aniketdivekar — APPROVED** [review]

File: tsconfig.json [hunk 3]
```diff
@@ -2,12 +2,13 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "target": "ES2021",
-    "rootDir": "src",
```
Comment (shyamvyas5) [inline tsconfig.json:5]: As we are changing the build directory, it's breaking the deployment flow as it creates `server.js` in different directory than we expect/used to be which is inside `build` directory, so we can run `node build/server.js`

**Comment by shyamvyas5** [comment]
And not just that we've to change command in `package.json` as `npm run start` command expects `server.js` to be inside `build` directory

**Comment by shyamvyas5** [comment]
Please keep the old flow and inside the same directory where it used to put the build before. Thanks.

File: tsconfig.json (hunk 3 above)
Comment (shyamvyas5) [reply tsconfig.json:5]: We've to change command in `package.json` as well [here](https://github.com/shardeum/archive-server/blob/23e6cbb6a32d32eebca5ee1a27f5b91b34160d98/package.json#L18) `npm run start` command expects `server.js` to be inside `build` directory

**Pull request merged**